app/ExposureNotificationContext.tsx

```diff
@@ -1,11 +1,39 @@
 /*global JSX*/
 import React, { createContext, useState } from 'react';
+import dayjs from 'dayjs';
 
 import * as ExposureNotifications from './exposureNotificationsNativeModule';
 
 export type ENAuthorizationStatus = 'authorized' | 'notAuthorized';
 
+type Posix = number;
```

Comment:
Is this used for the date? I thought typescript recognized a Date interface, but I could be wrong.

Reply:
it is! using posix a thing a picked up from working with elm, just keeping things in posix(time im milliseconds format) as to not need to deal with timezone stuff/minimize data payload size.  Once we figure out more about the en exposure data, it might make sense to choose a different format.